src/vfs.js

```diff
@@ -69,12 +69,16 @@ const pathToObject = path => ({
   ...typeof path === 'string' ? {path} : path
 });
 
+// Extract the mountpoint of a path
+const mountPoint = ({path}) => path.split(':/')[0];
+
 // Handles directory listing result(s)
 const handleDirectoryList = (path, options) => result =>
   Promise.resolve(result.map(stat => createFileIter(stat)))
-    .then(result => transformReaddir(pathToObject(path), result, capabilityCache, {
+    .then(result => transformReaddir(pathToObject(path), result, {
       showHiddenFiles: options.showHiddenFiles !== false,
-      filter: options.filter
+      filter: options.filter,
+      serverSorting: capabilityCache[mountPoint(pathToObject(path))] ? capabilityCache[mountPoint(pathToObject(path))].sort : false
```

Comment:
I'll defer to Anders' judgement on this, but I think this could stand to be broken up with new lines since it is so long.

Reply:
> I'll defer to Anders' judgement on this, but I think this could stand to be broken up with new lines since it is so long.

How about below code? @ajmeese7 @andersevenrud 

```js
const handleDirectoryList = (path, options) => result =>
  Promise.resolve(result.map(stat => createFileIter(stat)))
    .then(result => {
      let capability = capabilityCache[mountPoint(pathToObject(path))];
      return transformReaddir(pathToObject(path), result, {
        showHiddenFiles: options.showHiddenFiles !== false,
        filter: options.filter,
        serverSorting: capability.sort ? capability.sort : false
      });
    });
```

Reply:
@mahsashadi you can change:
```js
serverSorting: capability.sort ? capability.sort : false
```
To the following:
```js
serverSorting: capability.sort || false
```
To reduce the redundancy of it a bit.